Source/WebKit/Platform/spi/Cocoa/PDFKitSPI.h

```diff
@@ -162,14 +162,10 @@
 
 #endif
 
-#if HAVE(PDFSELECTION_ENUMERATE_RECTS_AND_TRANSFORMS)
-
 @interface PDFSelection (Staging_125426369)
 - (void)enumerateRectsAndTransformsForPage:(PDFPage *)page usingBlock:(void (^)(CGRect rect, CGAffineTransform transform))block;
 @end
 
-#endif
-
```

Comment:
While you're here, do you mind moving this staging declaration into the `!USE(APPLE_INTERNAL_SDK)` area? Under this interface: https://github.com/WebKit/WebKit/blob/6687e347f6f9a0538b3bb2902a530161a1201195/Source/WebKit/Platform/spi/Cocoa/PDFKitSPI.h#L91-L96